### DISCUSSION TITLE
Investigation of stuck IBC transactions

### DISCUSSION THREAD

**Discussion by KamuelBob** [akash-network/community#433] [Economics]
From previous times when IBC transfers weren't as user friendly in the Cosmos Ecosystem and the IBC module did not have the protections in place that it has today, a number of IBC transfers were made to expired clients that never properly failed and as a result the user funds have not been returned.

In particular, Keplr IBC transfers would default to channel-0 if no channel was entered and this was an expired channel to the Cosmos Hub.

https://www.mintscan.io/akash/relayers/channel-0/cosmos/channel-95

You can see at the above link that recent transactions attempted to this channel have failed, however a number of earlier transactions were reported as successful however were reported as never having arrived at the destination.

Is there a method such as migration of the expired channel (such as [recently done on the Chihuahaua blockchain](https://www.mintscan.io/chihuahua/proposals/65)) that would return the user funds?

**Comment by ghost** [comment]
Hi @KamuelBob 

Could you please provide us with at least a single TX ID of such IBC transfer so we can first and foremost - verify whether the IBC transfer has really stuck (i.e. never arrived nor returned back). ?

**Reply by KamuelBob** [reply]
PM sent.

**Reply by ghost** [reply]
after looking at the TX, it looks like the funds indeed stuck.

I think we should revive the expired channel as [suggested](https://www.mintscan.io/chihuahua/proposals/65)

Here is more info reg. this:
https://ibc.cosmos.network/main/ibc/proposals.html
